### DISCUSSION TITLE
XOR Bitwise Operator

### DISCUSSION THREAD

**Discussion by eduardomrodrigues** [fabiosvm/hook-lang#14] [Ideas]
I noticed that last merged "pull request" added bitwise operators, but not all of them. I missing XOR. Was that intentional? Normally, in C-style languages, the token "^" is used as operator.

**Comment by fabiosvm** [comment]
You are right. This bitwise operator is missing. I realized I was leaving it behind, perhaps because it was the least common. It will be implemented shortly.

**Comment by fabiosvm** [comment]
This feature has just been added with the pull request #16.
